benchmark/serumapi/output.go

```diff
@@ -195,6 +217,11 @@ func PrintSummary(runtime time.Duration, serumEndpoint string, solanaEndpoint st
 			serumTs := matchedTs[0]
 			solanaTs := matchedTs[1]
 
+			// sometimes only 1 of asks and bids are matched
```

Comment:
I am not sure I understand when this case would happen

Reply:
saw some instances where there was only an update on the orderbook for bid or ask for a particular slot, but not both. for example, I think if someone cancels their bid that would cause this kind of case